.buildkite/generator.jl

```diff
@@ -0,0 +1,15 @@
+include("utils.jl")
+
+pipeline = """
+steps:
+  - label: "test"
+    agents:
+      queue: "juliagpu"
```

Comment:
Note that Yggdrasil requires a very large amount of storage, I'm not sure it should eventually run on the same hardware as the rest of the ecosystem.

Reply:
Oh for sure, I just wanted something to start iterating with. The other issue is to solve caching.

So yes we will need a new queue, probably.